src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1444,6 +1448,19 @@ void Graph::EnforceBF16() {
     }
 }
 
+void Graph::EnableSparseWeightsForFC(float minSparseRate) {
```

Comment:
just `setMinSparseRate` - this is what the function actually do.

Reply:
changed, thanks